public/components/CampaignAnalytics/CampaignAnalytics.js

```diff
@@ -1,97 +0,0 @@
-import React, {PropTypes} from "react";
-import CampaignPerformanceSummary from "./Analytics/CampaignPerformanceSummary";
-import CampaignContentContributionPie from "./Analytics/CampaignContentContributionPie";
-import CampaignDailyUniquesChart from "./Analytics/CampaignDailyUniquesChart";
-import CampaignDailyTrafficChart from "./Analytics/CampaignDailyTrafficChart";
-import CampaignPagesCumulativeTrafficChart from "./Analytics/CampaignPagesCumulativeTrafficChart";
-import ContentTrafficChart from "./Analytics/ContentTrafficChart";
-import CampaignQualifiedChart from "./Analytics/CampaignQualifiedChart";
-import CampaignTrafficDriverStatsChart from "./Analytics/CampaignTrafficDriverStatsChart";
-
-class CampaignAnalytics extends React.Component {
-
-  isAnalysisAvailable(campaign) {
-    const analysableStatus = campaign.status === 'live' || campaign.status === 'dead';
-    return (analysableStatus && campaign.startDate && campaign.pathPrefix );
-  }
-
-  getLatestPageViews() {
-    if(this.props.campaignPageViews) {
-      return this.props.campaignPageViews.pageCountStats[this.props.campaignPageViews.pageCountStats.length - 1];
-    }
-
-    return undefined;
-  }
-
-  getLatestUniqueUsers() {
-    if(this.props.campaignDailyUniques) {
-      return this.props.campaignDailyUniques.dailyUniqueUsers[this.props.campaignDailyUniques.dailyUniqueUsers.length - 1];
-    }
-
-    return undefined;
-  }
-
-  render () {
-    if (!this.isAnalysisAvailable(this.props.campaign)) {
-      return null;
-    }
-
-    if (!this.props.campaignPageViews) {
-      return <div className="campaign-info__body">Loading... </div>;
-    }
-
-    return (
-      <div>
-        <div className="campaign-info__body">
-
-          <CampaignPerformanceSummary campaign={this.props.campaign}
```

Comment:
Do we not want to keep CampaignPerformance? Just change the graph? Or maybe your plan is to completely re-write the graph?